src/main.rs

```diff
@@ -160,13 +169,18 @@ fn main() -> Result<()> {
         .or_else(|_| EnvFilter::try_from(&option.verbose))
         .or_else(|_| EnvFilter::try_new("trace"))
         .unwrap();
-    tracing_subscriber::fmt().with_env_filter(env_filter).init();
+    tracing_subscriber::fmt().with_writer(io::stderr).with_env_filter(env_filter).init();
 
     let mount_injector = inject(option.clone())?;
```

Comment:
Can we collect this `Err` and report it through `ping`? (or maybe `ping` is not a suitable name for this complicate situation.)

Reply:
Sorry I don't quite follow, do you mean the `Err` that may be generated by `let mount_injector = inject(option.clone())?;` In that case toda shall exit (paniced) at once and there will be no rpc server anymore. I don't know if it would be a good idea to hold it up for another several seconds.